hypothesis-python/src/hypothesis/internal/conjecture/dfa/__init__.py

```diff
@@ -16,6 +16,10 @@
 import threading
 from collections import deque
 
+from hypothesis.internal.reflection import proxies
+
+INFINITY = float("inf")
```

Comment:
Is this just `math.inf` nowadays? I think it dates back to Python 3.5.

Reply:
Oh, I didn't know that existed. Good knowledge, thanks!